src/app/item-page/simple/field-components/preview-section/file-description/file-description.component.ts

```diff
@@ -45,4 +56,9 @@ export class FileDescriptionComponent {
     const imgElement = event.target as HTMLImageElement;
     imgElement.src = this.MIME_TYPE_IMAGES_PATH + this.MIME_TYPE_DEFAULT_IMAGE_NAME;
   }
+
+  isArchive(format: string): boolean {
+    return format === 'application/zip' || format === 'application/x-tar';
```

Comment:
this is unfortunate, can we find out if the preview is available using metadata? or other? this is strong and uneeded coupling

Reply:
What? I do not understand your point. This only check if the preview, which already exist is ZIP or TAR for proper displaying.